### PR TITLE
Add setup instructions for Anaconda Python distribution

### PR DESCRIPTION
**Pull request by gojira**
These instructions show how to install tensorflow with the Anaconda Python distribution.
1. Use 'conda env' instead of virtualenv
2. Add --ignore-installed flag for pip install to avoid warning about easy_install

### CODE REVIEW

**Comment by gunan** [comment]
Can one of the admins verify this patch?

File: tensorflow/g3doc/get_started/os_setup.md [hunk 1]
```diff
@@ -175,6 +178,93 @@ $ source ~/tensorflow/bin/activate.csh  # If using csh.
...
+
+# Python 3.5
+$ conda create -n tensorflow python=3.5
+```
+
+Activate the environment and use pip to install TensorFlow inside it.
+Use the `--ignore-installed` flag to prevent errors about `easy_install`.
+
+```bash
+$ source activate tensorflow
```
Comment (martinwicke) [inline tensorflow/g3doc/get_started/os_setup.md]: Will this work from any directory? Does the preceding conda command put you in the right directory?

File: tensorflow/g3doc/get_started/os_setup.md [hunk 2]
```diff
@@ -175,6 +178,93 @@ $ source ~/tensorflow/bin/activate.csh  # If using csh.
...
+(tensorflow)$  # Your prompt should change
+
+# Ubuntu/Linux 64-bit, CPU only:
+(tensorflow)$ pip install --ignore-insalled --upgrade https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.8.0rc0-cp27-none-linux_x86_64.whl
```
Comment (martinwicke) [inline tensorflow/g3doc/get_started/os_setup.md]: insalled => installed

File: tensorflow/g3doc/get_started/os_setup.md [hunk 3]
```diff
@@ -175,6 +178,93 @@ $ source ~/tensorflow/bin/activate.csh  # If using csh.
...
+
+# Ubuntu/Linux 64-bit, GPU enabled:
+(tensorflow)$ pip install --ignore-installed --upgrade https://storage.googleapis.com/tensorflow/linux/gpu/tensorflow-0.8.0rc0-cp27-none-linux_x86_64.whl
+
+# Mac OS X, CPU only:
+(tensorflow)$ pip install --ignore-installed --upgrade https://storage.googleapis.com/tensorflow/mac/tensorflow-0.8.0rc0-py2-none-any.whl
+```
+
+and again for python3:
```
Comment (martinwicke) [inline tensorflow/g3doc/get_started/os_setup.md]: "Python 3"

File: tensorflow/g3doc/get_started/os_setup.md [hunk 4]
```diff
@@ -175,6 +178,93 @@ $ source ~/tensorflow/bin/activate.csh  # If using csh.
...
+```bash
+(tensorflow)$ source deactivate
+
+$  # Your prompt should change back
+```
+
+To use TensorFlow later you will have to activate the conda environment again:
+
+```bash
+$ source activate tensorflow
```
Comment (martinwicke) [inline tensorflow/g3doc/get_started/os_setup.md]: Again: Independent of current directory?

**Comment by martinwicke** [comment]
I left a few comments. Thanks for writing this up!

File: tensorflow/g3doc/get_started/os_setup.md (hunk 1 above)
Comment (gojira) [inline tensorflow/g3doc/get_started/os_setup.md]: Yes, conda places environment definitions in a global Anaconda environment folder. You can invoke 'source activate tensorflow' from any directory, start python, and import tensorflow.

File: tensorflow/g3doc/get_started/os_setup.md (hunk 2 above)
Comment (gojira) [inline tensorflow/g3doc/get_started/os_setup.md]: I will fix that

File: tensorflow/g3doc/get_started/os_setup.md (hunk 3 above)
Comment (gojira) [inline tensorflow/g3doc/get_started/os_setup.md]: I can fix that - note that I copied the pattern here from the virtualenv description and that's where the python3 comes from.

File: tensorflow/g3doc/get_started/os_setup.md (hunk 4 above)
Comment (gojira) [inline tensorflow/g3doc/get_started/os_setup.md]: Yes see my comment from the other.  You can activate, run python, and import tensorflow anywhere.

File: tensorflow/g3doc/get_started/os_setup.md (hunk 2 above)
Comment (gojira) [inline tensorflow/g3doc/get_started/os_setup.md]: I commited this update

File: tensorflow/g3doc/get_started/os_setup.md (hunk 3 above)
Comment (gojira) [inline tensorflow/g3doc/get_started/os_setup.md]: I committed this change

**Comment by gojira** [comment]
I committed fixes to your comments.  Thanks for quick review!

**Comment by gojira** [comment]
Thanks!

**Pull request merged**